angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.stories.ts

```diff
@@ -116,9 +121,7 @@ export const informational = () => ({
   `,
 });
 
-informational.story = {
-  parameters: {
-    docs: { iframeHeight: 400 },
-    jest: ['sprk-dropdown.component'],
-  },
+informational.parameters = {
+  docs: { iframeHeight: 400 },
```

Comment:
This doesn't appear to be getting applied either.

Reply:
@guern1kn , (same as above ) it doesn't get applied with inline story rendering. I have a note up in the PR description about it. I left them in the code because it would be something we want to have and if SB adds it in a patch fix then we don't have to re-add it all.